launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -27,6 +27,7 @@
   <arg name="use_pointcloud_container" default="false" description="use pointcloud container for detection preprocessor"/>
   <arg name="container_name" default="pointcloud_container"/>
   <arg name="use_validator" default="true" description="use obstacle_pointcloud based validator"/>
+  <arg name="score_threshold" default="0.45"/>
```

Comment:
@yukke42 Can you check this value?

Reply:
(I think it is default value of CenterPoint now.)
https://github.com/autowarefoundation/autoware.universe/blob/main/perception/lidar_centerpoint/launch/lidar_centerpoint.launch.xml#L8

Reply:
Yes, It's the default value of lidar_centerpoint.